### DISCUSSION TITLE
Welcome to WhiteMatterTech Discussions!

### DISCUSSION THREAD

**Discussion by RobertDWhite** [RobertDWhite/WhiteMatterTech#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Helderwk** [comment]
Hello Robert, I read your post "How to Add VLAN Segmentation for HomeKit IoT Devices with Unifi", very good hints. As see you as a Unifi deep knowledge, I would like to know whether Unifi devices are alredy prepared or will be supporting the Matter Standard? Are you aware about it?

**Reply by RobertDWhite** [reply]
@Helderwk 
Sorry for the delay. The network gear supports the Matter Standard insofar as the Matter Standard works on the network gear. For the Protect cameras and IoT devices, I would be surprised if we saw an integration with Matter anytime in the near future, as the Protect cameras still do not natively support HomeKit. With that said, there is a feature request open for Matter support here: https://community.ui.com/questions/Wishlist-Matter-support/a7bc072b-7cc2-4ada-a8be-b6aadb22a987?page=1
And Ubiquiti Inc. is indeed listed as an adopter for the protocol: https://csa-iot.org/members/adopters/

One can only hope!

**Comment by Helderwk** [comment]
Thank you Robert for your kind reply!

Em qua., 19 de abr. de 2023 às 03:26, Robert ***@***.***>
escreveu:
